scripts/main/contextMenu.js

```diff
@@ -548,15 +649,22 @@ contextMenu.move = function (IDs, e, callback, kind = "UNSORTED", display_root =
 	});
 };
 
+/**
+ * @param {string} photoID
+ * @param {jQuery.Event} e
+ *
+ * @returns {void}
+ */
 contextMenu.sharePhoto = function (photoID, e) {
 	// v4+ only
-	if (photo.json.hasOwnProperty("share_button_visible") && photo.json.share_button_visible !== "1") {
+	// TODO: IMHO `photo.json` always jas the property `is_share_button_visible`
```

Comment:
Yep, as implied by the comment just above. The attribute wasn't there in v3. Code can be removed now.